cristin-import/src/main/java/no/unit/nva/cristin/CristinObject.java

```diff
@@ -0,0 +1,29 @@
+package no.unit.nva.cristin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.LocalDate;
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+public class CristinObject {
```

Comment:
The object that will hold the information from Cristin. I think this is going to be a manual mapping between the json fields and the java fields